### PR TITLE
PR Deploy GH action

### PR DESCRIPTION
**Pull request by AtherBilal**
This will deploy PR to S3 so that we can test them before merging

### CODE REVIEW

File: app/Snackbar.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ function SimpleSnackbar(props) {
     }
     setOpen(false)
   }
-  let message = 'Welcome to ol-kit!'
+  let message = 'Welcome to HELL!!!!'
```
Comment (stazrad) [inline app/Snackbar.js:18]: lol wut

Comment (ntatko) [reply app/Snackbar.js:18]: um

File: .github/workflows/PR-deploy-to-s3.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-east-2
+
+    - name: Install npm deps
+      run: |
```
Comment (ntatko) [inline .github/workflows/PR-deploy-to-s3.yml:26]: Why are you piping this? One-liners don't need this.

**Comment by ntatko** [comment]
https://github.com/MonsantoCo/ol-kit/issues/24

**Pull request closed without merging**